mm/2s2h/DeveloperTools/WarpPoint.cpp

```diff
@@ -107,6 +108,17 @@ void RegisterWarpPoint() {
 }
 
 void RenderWarpPointSection() {
+    bool skipToFileSelect = CVarGetInteger("gEnhancements.Cutscenes.SkipToFileSelect", 0);
+    UIWidgets::CVarCheckbox(
+        "Boot to Warp Point on Launch", WARP_POINT_CVAR "BootToWarpPoint",
+        UIWidgets::CheckboxOptions({ { .disabled = CVarGetInteger("gEnhancements.Cutscenes.SkipToFileSelect", 0),
+                                       .disabledTooltip = "Incompatible with Skip to File Select enhancement" } })
```

Comment:
Windows does not like trying to use this uint32_t as a bool:
```
D:\a\2ship2harkinian\2ship2harkinian\mm\2s2h\DeveloperTools\WarpPoint.cpp(114): error C2397: conversion from 'int32_t' to 'bool' requires a narrowing conversion
```

Further, `skipToFileSelect` is never used. Maybe change this to:


```suggestion
    bool skipToFileSelect = (bool)CVarGetInteger("gEnhancements.Cutscenes.SkipToFileSelect", 0);
    UIWidgets::CVarCheckbox(
        "Boot to Warp Point on Launch", WARP_POINT_CVAR "BootToWarpPoint",
        UIWidgets::CheckboxOptions({ { .disabled = skipToFileSelect, 0),
```

Reply:
woops, this was my intention but must have forgotten to finish